code/modules/mob/living/simple_animal/bot/secbot.dm

```diff
@@ -196,8 +196,8 @@
 /mob/living/simple_animal/bot/secbot/attackby(obj/item/W, mob/user, params)
 	..()
 	if(istype(W, /obj/item/weldingtool) && user.a_intent != INTENT_HARM) // Any intent but harm will heal, so we shouldn't get angry.
-		return
-	if(!istype(W, /obj/item/screwdriver) && (W.force) && (!target) && (W.damtype != STAMINA) ) // Added check for welding tool to fix #2432. Welding tool behavior is handled in superclass.
+		return // Added check for welding tool to fix #2432. Welding tool behavior is handled in superclass.
```

Comment:
don't reference issue numbers on the github in comment, instead describe what this is fixing rather than outsourcing it

Reply:
This was made by someone else, I just had to move the comment because it was less useful than the new comment I added. Want me to remove this comment?